extension/cgroupruntimeextension/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Cgroup Go runtime extension
+
+In order to contribute to this extension, it might be useful to have a working local setup.
+
+## Testing
+
+To run the integration tests locally for this extension, you can follow theses steps in a Linux environment.
+
+Inside the extension folder, start a privileged docker container and share the code with the container
+
+```bash
+cd extension/cgroupruntimeextension
+docker run -ti --privileged --cgroupns=host -v $(pwd):/workspace -w /workspace debian:bookworm-slim
+```
+
+Install Go and gcc to run the integration test
```

Comment:
Instead of pinning a specific Go version as a copy and paste command, wdyt if we just add it as a complementary example to:

```suggestion
Install the [Go version](https://go.dev/dl/) specified in the extension's [go.mod](./go.mod) and the GCC compiler to run the integration test. The following is an example command for Go `1.23.4` in and `amd64` system:
```